app/src/main/java/com/codingblocks/cbonlineapp/dashboard/DashboardActivity.kt

```diff
@@ -85,6 +85,10 @@ class DashboardActivity : BaseCBActivity(),
         toggle.syncState()
         dashboardNavigation.setNavigationItemSelectedListener(this)
         dashboardBottomNav.setOnNavigationItemSelectedListener(this)
+        if(vm.isLoggedIn==false){
```

Comment:
Instead of this make this inVisible be default like inbox and enable on login only